config/services.yaml

```diff
@@ -4,6 +4,7 @@ imports:
     - { resource: parameters_addons.yaml }
 
 parameters:
+    env(APP_SECRET): '%secret%'
```

Comment:
this is the trick to preserve BC! if the APP_SECRET env var is not set, it will use the value from the `%secret%` parameter!